python/cuml/common/handle.pyx

```diff
@@ -48,8 +49,8 @@ cdef class Handle:
 
         # call ML algos here
 
-        # final synchronization of all work launched/dependent on this stream
-        stream.sync()
+        # final sync of all work launched in the stream of this handle
+        handle.sync()
```

Comment:
Same as with `base.sync()`: What is the motivation to provide this API?

Reply:
This is done now and the `base.sync` is being discussed [here](https://github.com/rapidsai/cuml/pull/475#discussion_r275665989). Resolving this one on that behalf.